qa/rpc-tests/p2p-timeouts.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+# Copyright (c) 2016 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+from test_framework.mininode import *
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import *
+from time import sleep
+
```

Comment:
I think we should try to use a consistent PEP8 convention for frontmatter:

- shebang
- copyright notice
- module docstring
- standard library imports
- project imports

(see https://github.com/bitcoin/bitcoin/pull/9577). Can I convince you to do the same here?